adam/curriculum/phase1_curriculum.py

```diff
@@ -318,22 +318,37 @@ def _make_object_on_object_curriculum() -> _Phase1InstanceGroup:
 
 def _make_object_in_other_object_curriculum() -> _Phase1InstanceGroup:
     object_ = object_variable("object_0", banned_properties=[IS_BODY_PART])
+    liquid = object_variable("liquid_0", required_properties=[LIQUID], banned_properties=[IS_BODY_PART])
     containing_object = object_variable(
         "object_1", required_properties=[HOLLOW], banned_properties=[IS_BODY_PART]
     )
-    situation_template = Phase1SituationTemplate(
+    solid_template = Phase1SituationTemplate(
         object_variables=[object_, containing_object],
         constraining_relations=[Relation(BIGGER_THAN, containing_object, object_)],
         asserted_always_relations=[inside(object_, containing_object)],
     )
+    liquid_template = Phase1SituationTemplate(
+        object_variables=[object_, containing_object],
+        asserted_always_relations=[inside(object_, containing_object)],
```

Comment:
Shouldn't this be `liquid`, not `object_`, both here and in the object variables?

Reply:
Yes, realized after pushing, fixing now. Interestingly it produced liquids anyway.